addons/hr_expense/models/account_move.py

```diff
@@ -53,10 +53,11 @@ def _compute_payment_state(self):
     def _compute_needed_terms(self):
         # EXTENDS account
         # We want to set the account destination based on the 'payment_mode'.
+        # Also, expense' account moves are expressed in the company currency.
```

Comment:
it's not the case when a vendor bill is created, right?

Reply:
yes -- the move will always be created in company currency because there might be expenses with different currencies inside the same expense sheet.